gettsim/parameters/ges_rente.yaml

```diff
@@ -730,30 +730,84 @@ regelaltersgrenze:
   description:
     de:
       Stückweise lineare Funktion, die die Regelaltersgrenze angibt, bei der das Individuum mit seinem vollen Rentenanspruch in Rente
-      gehen kann. Geht die Person früher oder später in Rente, ist der Zugangsfaktor und damit der Rentenanspruch höher oder niedriger.
+      gehen kann. Geht die Person früher oder später in Rente, ist der Zugangsfaktor und damit der Rentenanspruch höher oder niedriger,
+      wenn keine Ausnahmeregelungen erfüllt sind.
     en:
       Piecewise linear function returning the regelaltersgrenze at which the agent is allowed to get pensions with his full
       claim. If the agent retires earlier or later, the Zugangsfaktor and therefore
-      the pension claim is higher or lower.
+      the pension claim is higher or lower, unless special conditions apply.
   reference: § 35 Satz 2 SGB VI
   note:
     de: >-
       Diese Funktion ist hier ab 2002 angegeben. Vor 2002 ist das Steuersystem nicht
```

Comment:
The parameters here should reflect the normal retirement age 65 for cohorts before 1947 and for later cohorts adapted according to the reform. 
To my knowledge the "Regelaltersgrenze" as such is not different for women. 
There used to be the "pension for women" (up to cohort 1951) though, which is not implemented yet. (As many other pathways to retirement without deductions "Rente für langjährig Versicherte", "Rente für besonders langjährig Versicherte". ) I would prefer treating women of the affected cohorts as special cases, since after 1952 the "pension for women" was abolished.